.github/ISSUE_TEMPLATE/new-repo.md

```diff
@@ -21,18 +21,18 @@ Sponsoring WG:
 
 ## Actions to fulfill
 
-This area is used for the TOC to track the [repo creation
-process](/mechanics/CREATING-A-SANDBOX-REPO.md)
-
+This area is used for the TOC to track the [repo creation process](/mechanics/CREATING-A-SANDBOX-REPO.md).
+The _requestor_ should perform the steps listed below and cross out the checkmarks when done.
+The TOC is involved only in the **TOC Gate** steps.
 
 - [ ] Add this issue to the [TOC project board](https://github.com/orgs/knative/projects/9) for review.
```

Comment:
As you mention there's been some discussion of who can do this and how, it's worth clarifying that here/now.

I believe what @grantr and @n3wscott found was that anyone in the org could add stuff **from the project board UI**, but couldn't via the gear wheel on the issue itself.

Reply:
Hmm, I could add https://github.com/knative/client-contrib/issues/75 without issues from the issue itself to the project board in community. But maybe because I have some special karma that not everyone has ?

![image](https://user-images.githubusercontent.com/99080/92618912-25805a00-f2c1-11ea-9e45-2c9a108eeb44.png)

Reply:
Yeah, you are an editor on that repo.  If folks are creating the issue in community, then virtually nobody is an editor.

Reply:
An explanatory sentence has been added to the template in the meantime.